src/QMCWaveFunctions/TrialWaveFunction.h

```diff
@@ -613,7 +633,12 @@ class TrialWaveFunction
   // helper function for extracting a list of laplacian from a list of TrialWaveFunction
   static RefVector<ParticleSet::ParticleLaplacian> extractLRefList(
       const RefVectorWithLeader<TrialWaveFunction>& wf_list);
+
+  std::unique_ptr<TWFFastDerivWrapper> twf_fastderiv_;
+  // cache to avoid re-initializing the wrapper with the same config
+  mutable const ParticleSet* twf_fastderiv_last_P_ = nullptr;
+  mutable std::size_t twf_fastderiv_last_Zcount_   = 0;
```

Comment:
Very likely unsafe optimization. Please avoid them unless shown absolutely necessary. Remove both mutable please.

Reply:
I meant also remove `twf_fastderiv_last_P_` and `twf_fastderiv_last_Zcount_`